### ISSUE TITLE
aria-posinset should be 1-based, not 0-based

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1053]
This will also require rewriting a lot of the tests, which rely on aria-posinset heavily.

**Comment by nolanlawson** [comment]
I should have read the spec more closely: https://www.w3.org/TR/wai-aria-1.1/#aria-posinset

> Authors MUST set the value for aria-posinset to an integer greater than or equal to 1,
